parallaxinc/solo
# Solo 356 - Get graph settings directly from blocks

Begins addressing #356 
 - Fixes a CSS error that is styling the lines on the graph incorrectly (I think I broke this a while back, and have been meaning to revert/fix it)
 - Pulls the code that sets the chartist.js object's options/settings using info from the blocks into its own function
 - Transitions the graphing (and terminal) code away from parsing the generated propc for its settings and labels, and instead grab them directly from the block fields.

To test:
 - Create a program that graphs.  Ensure graph opens, functions, and renders correctly when the prop has loaded.
 - Create a program that prints to the terminal.  Ensure the terminal opens when the prop has loaded.
